composer.json

```diff
@@ -27,15 +27,15 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-    "zendframework/zend-stdlib": "^2.7 || ^3.0",
-    "zfr/rbac": "~2.0"
+    "php": "^7.0",
+    "zendframework/zend-servicemanager": "^3.0.3",
+    "zendframework/zend-stdlib": "^3.0"
```

Comment:
@prolic not sure it's necessary to be that aggressive and support only 3.0 is this just because we can have few factories  more clean? stdlib I think does not require any changes at all. Not all packages allows 3.0 yet and that's problem.